reference/providers/kubernetes-v2/index.md

```diff
@@ -114,8 +114,117 @@ Kubernetes supports. Also the Kubernetes extension mechanisms&mdash;called
 it easy to build new types of resources, and Spinnaker accomodates that by
 making it simple to extend Spinnaker to support a user's CRDs.
 
+It is worth noting that the resource mapping exists primarily to render
+resources in the UI according to Spinnaker conventions, it does not affect how
+resources are deployed or managed. Instead, this is determined by the following
+properties:
+
+* __Versioning__
+
+  If a resource is "versioned", it will always be deployed with a new sequence
+  number `vNNN`. This is important for resources like ConfigMaps and
+  ReplicaSets, which don't have their own builtin update policy like
+  Deployments or StatefulSets do. Whatever policy exists can be overriden
+  during a deploy manifest stage.
+
+* __Operations__
+
+  There are several operations that can be implemented by each kind:
+
+  * _Deploy:_ Can this resource be deployed?
+  * _Delete:_ Can this resource be deleted?
+  * _Scale:_  (Workloads only) Can this resource be scaled to a desired replica
+    count?
+  * _Undo Rollout:_ (Workloads only) Can this resource be rolled back/forward
+    to an existing revision?
+  * _Pause Rollout:_ (Workloads only) When rolling out, can the rollout be
+    stopped?
+  * _Resume Rollout:_ (Workloads only) When a rollout is paused, can it be
+    started again?
+
+* __Stability__
+
+  This describes under what conditions is this kind considered stable after a
+  new `spec` has been submitted.
+
 ## Workloads ≈ Server Groups
 
+Anything classified as a Spinnaker Server Group will be rendered on the
```

Comment:
Feels like the information below would be easier to consume if presented via a table.

Reply:
Done